README.md

```diff
@@ -112,4 +110,4 @@ Please file feature requests and bugs at the [issue tracker][tracker].
 [sha1-obj]: https://www.dartdocs.org/documentation/crypto/latest/crypto/sha1.html
 [sha256-obj]: https://www.dartdocs.org/documentation/crypto/latest/crypto/sha256.html
 [startChunkedConversion]: https://www.dartdocs.org/documentation/crypto/latest/crypto/Hash/startChunkedConversion.html
-[tracker]: https://github.com/dart-lang/crypto/issues
\ No newline at end of file
+[tracker]: https://github.com/dart-lang/crypto/issues
```

Comment:
Apparently the GitHub "propose file changes" feature added a trailing newline to this file...